cms/djangoapps/contentstore/views/entrance_exam.py

```diff
@@ -270,6 +271,14 @@ def _delete_entrance_exam(request, course_key):
         }
         CourseMetadata.update_from_dict(metadata, course, request.user)
 
+    # Remove any/all Entrance Exam graders from the course
+    grading_model = CourseGradingModel.fetch(course_key)
+    graders = grading_model.graders
+    # Clean up any pre-existing Entrance Exam grader cruft
+    for i, grader in enumerate(graders):
+        if grader['type'] == 'Entrance Exam':
```

Comment:
Is this type name always generated? Might want to give it a const name somewhere and refer to that here.